docs/introduction-to-smart-contracts.rst

```diff
@@ -115,58 +108,56 @@ registering with username and password — all you need is an Ethereum keypair.
 
 This contract introduces some new concepts, let us go through them one by one.
 
-The line ``address public minter;`` declares a state variable of type address
```

Comment:
Please do not change the line breaks here.

Reply:
This was somewhat intentional, to break up the spacing and thus flow of a lot of explanation for someone new to Solidity, breaking each concept into a new paragraph, making it easier to scan through them one by one. White space helps with easing this flow. Unless I am misunderstanding what you mean?

Reply:
I mean the line breaks in the source code, not in the rendered output - it is hard to review this way.